src/Lykke.Service.BlockchainSettings.Services/BlockchainValidationService.cs

```diff
@@ -23,14 +26,14 @@ public async Task<bool> ValidateHotwalletAsync(string apiUrl, string type, strin
             {
                 //TODO: WAIT FOR NEW IBlockchainApiClient on 2.1 framework (httpClientFactory will be there)
                 //DIRTY HACK:
-                using (var client = new Lykke.Service.BlockchainApi.Client.BlockchainApiClient(_log, apiUrl, 3))
+                using (var client = new BlockchainApiClient(_logFactory, apiUrl, 3))
                 {
                     isAddressValid = await client.IsAddressValidAsync(address);
                 }
             }
             catch (Exception e)
             {
-                _log.WriteInfo(nameof(ValidateHotwalletAsync), $"{apiUrl}, {type}, {address}", "Could not check validity");
+                _log.Info(nameof(ValidateHotwalletAsync), $"{apiUrl}, {type}, {address}", "Could not check validity");
```

Comment:
This was before your modification, but exeption just swalloed here - it's bad practice. Exception should be either logged or rethrowed